### PR TITLE
Fix ericsson MINI LINK wifi sensors

### PR DESCRIPTION
**Pull request by erotel**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [ X] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

**Comment by erotel** [comment]
Hello. Does anyone know why it reports me error `1) LibreNMS \ Tests \ YamlTest :: testDiscoveryDefinitionSchema
ericsson-ml.yaml does not validate. Violations:
[modules.sensors.state.data [1]] NULL value found, but an object is required`
I did again `./scripts/collect-snmp-data.php -h <yourDeviceId>` and `./scripts/save-test-data.php -o ericsson-ml` but I still get an error at startup `./scripts/pre- commit.php`
Otherwise the code works, the graph is ok.

File: includes/definitions/discovery/ericsson-ml.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+            - { value: 28, generic: 0, graph: 1, descr: '1024Qam' }
+            - { value: 29, generic: 0, graph: 1, descr: '1024QamLight' }
+            - { value: 30, generic: 0, graph: 1, descr: '1024QamStrong' }
+            - { value: 31, generic: 0, graph: 1, descr: '2048Qam' }
+            - { value: 32, generic: 0, graph: 1, descr: '2048QamLight' }
+            - { value: 33, generic: 0, graph: 1, descr: '2048QamStrong' }
+            - { value: 34, generic: 0, graph: 1, descr: '4096Qam' }
+            - { value: 35, generic: 0, graph: 1, descr: '4096QamLight' }
+            - { value: 36, generic: 0, graph: 1, descr: '4096QamStrong' }
+        -
```
Comment (Jellyfrog) [inline includes/definitions/discovery/ericsson-ml.yaml]: delete this line

Comment (erotel) [reply includes/definitions/discovery/ericsson-ml.yaml]: Thank you for your help

**Comment by murrant** [comment]
This pull request has been mentioned on **LibreNMS Community**. There might be relevant details there:

https://community.librenms.org/t/v1-56-release-changelog-september-2019/9671/1

**Pull request merged**